client/reader/tag-stream/main.jsx

```diff
@@ -26,7 +26,6 @@ class TagStream extends Component {
 	static propTypes = {
 		encodedTagSlug: PropTypes.string,
 		decodedTagSlug: PropTypes.string,
-		followSource: PropTypes.string.isRequired,
```

Comment:
This was causing a PropType warning in development, and requiring this prop doesn't seem to be relevant any more.